app/controllers/passwordless/application_controller.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Passwordless
+    # Base for Passwordless controllers
+    class ApplicationController < ::ApplicationController
```

Comment:
Same question as below - why does this bit need to be in our application code? The Passwordless should add this behind the scenes.